### PR TITLE
feat: add tarina tvl

### PR DESCRIPTION
**Pull request by Braveheartforyou**
##### Twitter Link:
https://twitter.com/tarinaexchange

##### Website Link:
https://www.tarina.exchange/

##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):
https://raw.githubusercontent.com/tarinaexchange/tarina-tokenlists/main/logos/0x0b1b00d9e7d9104d4ef0d36e9854526ac65e70f0/logo.png

##### Current TVL:

169.75 k 

##### Chain:

AVAX 43114

##### Short Description (to be shown on DefiLlama):
Tarina allows low cost, near 0 slippage trades on uncorrelated or tightly correlated assets. An revolutionanew Automated Market Maker (AMM) on the Avalanche Network. Low cost stable coin swaps,Transaction fee is only 0.01%.

##### Token address and ticker if any:
https://snowtrace.io/address/0x0b1b00d9e7d9104d4ef0d36e9854526ac65e70f0#code

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Dexes

##### forkedFrom (Does your project originate from another project):
Solidly

##### methodology (what is being counted as tvl, how is tvl being calculated):
https://snowtrace.io/address/0x0b1b00d9e7d9104d4ef0d36e9854526ac65e70f0#code

### CODE REVIEW

File: projects/tarina/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+const { request, gql } = require("graphql-request");
```
Comment (g1nt0ki) [inline projects/tarina/index.js:1]: Thanks for the PR, sorry but we really prefer on chain calls over graph calls, can you change it to this instead?


    const { calculateUsdUniTvl } = require('../helper/getUsdUniTvl');

    module.exports = {
    misrepresentedTokens: true,
    methodology: 'The Tarina subgraph and the Tarina factory contract address are used to obtain the balance held in every LP pair.',
    avalanche:{
        tvl: calculateUsdUniTvl(
            "0xb334a709dd2146caced08e698c05d4d22e2ac046",
            "avax",
            "0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7",
            [
            "0xB97EF9Ef8734C71904D8002F8b6Bc66Dd9c48a6E", //USDC
            "0xc7198437980c041c805a1edcba50c1ce5db95118", //usdte
            "0xa7d7079b0fead91f3e65f86e8915cb59c1a4c664", //usdce
            ],
            "wrapped-avax"
        )
    },
    start: 1650124800, // 17th-April-2022
    }

**Review by g1nt0ki — CHANGES_REQUESTED** [review]

**Review by Braveheartforyou — COMMENTED** [review]
edit tarina tvl

**Comment by Braveheartforyou** [comment]
Didn't see tarina on the https://defillama.com/chain/Avalanche @g1nt0ki

**Comment by g1nt0ki** [comment]
@Braveheartforyou we still need to manually add some metadata on the backend, give us a day :)

**Comment by Braveheartforyou** [comment]
> @Braveheartforyou we still need to manually add some metadata on the backend, give us a day :)

ok! tks！

**Pull request merged**